IdentityCore/src/cache/accessor/MSIDAccountCredentialCache.m

```diff
@@ -429,4 +431,39 @@ - (BOOL)saveWipeInfoWithContext:(nullable id<MSIDRequestContext>)context
     return [_dataSource saveWipeInfoWithContext:context error:error];
 }
 
+// Writing metadata
+- (BOOL)saveAppMetadata:(nonnull MSIDAppMetadataCacheItem *)metadata
+            context:(nullable id<MSIDRequestContext>)context
+              error:(NSError * _Nullable * _Nullable)error
+{
+    assert(metadata);
+    
+    MSID_LOG_VERBOSE(context, @"Saving app's metadata with clientId %@", metadata.clientId);
+    MSID_LOG_VERBOSE_PII(context, @"Saving app's metadata %@", metadata);
+    
+    MSIDAppMetadataCacheKey *key = [[MSIDAppMetadataCacheKey alloc] initWithClientId:metadata.clientId
+                                                                         environment:metadata.environment
+                                                                         generalType:MSIDAppMetadataType];
+    key.familyId = metadata.familyId;
```

Comment:
nit: Why don't you pass familyID in the init method?

Reply:
done